neurobooth_terra/dataflow.py

```diff
@@ -407,7 +406,10 @@ def delete_files(db_table, target_dir, suitable_dest_dir1, suitable_dest_dir2,
     # but could be an alternate source as well), is_deleted is False (because redundancy),
     # is_finished is True - i.e. copied successfully with hash check, and age is older
     # than 30 days
-    where = f"(position('{suitable_dest_dir1}' in dest_dirname)>0 OR position('{suitable_dest_dir2}' in dest_dirname)>0) "
+    where = ''
+    for vol in volumes:
```

Comment:
It's probably my limited exposure to python idioms, but I find this code complex relative to what it's doing. (TBH, I'm not entirely sure what it's doing.) It might have benefited from adding a couple statements and standard conditionals

Reply:
Maybe more clear if splitting into multiple steps:
```
volume_conditions = [f"position('{vol}' in dest_dirname) > 0" for vol in volumes]
where = '(' + ' OR '.join(volume_conditions) + ') '
```

Reply:
I went with the volume_conditions split. Thanks!